src/vendor-js/eth-eip712-util/index.d.ts

```diff
@@ -2,6 +2,10 @@ const module: {
   hashForSignTypedDataLegacy: (params: { data: any }) => Buffer
   hashForSignTypedData_v3: (params: { data: any }) => Buffer
   hashForSignTypedData_v4: (params: { data: any }) => Buffer
+  TypedDataUtils: {
+    sanitizeData: (data: any) => any,
+    hashStruct: (primaryType: string, data: any, types: any, useV4: any) => buffer
```

Comment:
```suggestion
    hashStruct: (primaryType: string, data: any, types: any, useV4: any) => Buffer
```
i don't think lowercase "buffer" is a thing